dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -152,6 +157,45 @@ public void testBom()
                 + "/ com.google.appengine:appengine-api-1.0-sdk:1.9.71 (provided)");
   }
 
+  @Test
+  public void testBom_java11()
+      throws IOException, RepositoryException, TransformerException, XMLStreamException {
+    // The number of linkage errors differ between Java 8 and Java 11 runtime.
+    Assume.assumeThat(System.getProperty("java.version"), StringStartsWith.startsWith("11."));
```

Comment:
maybe greater than or equal to 11?

Reply:
Updated. Yes, I want this test fail when we try next LTS JDK changes something related to linkage errors.